### DISCUSSION TITLE
Enable Border for Telescope

### DISCUSSION THREAD

**Discussion by Steven314** [NvChad/NvChad#2472] [Q&A] [answered]
I am attempting to enable the border for telescope.  My understanding (from [here](https://www.reddit.com/r/neovim/comments/168qxto/enable_border_in_telescope_plugin_nvchad/)) is that I need to insert `telescope = { style = "bordered" },` into the `M.ui` block of chadrc.lua in the custom folder. This however does not have any effect.
Other than the attempted change to the border, the setup is entirely stock to NvChad's default config that is installed when responding `Y` to the (Y/N) question upon installing NvChad.

The images below are taken from [here](https://nvchad.com/docs/features#telescope_nvim).
## Desired Effect
![image](https://github.com/NvChad/NvChad/assets/27896672/6960b6ec-cf05-45d5-9cb3-3048058ac41f)

## Current Status
![image](https://github.com/NvChad/NvChad/assets/27896672/e7dd5ba8-41ae-4d9d-ba26-110f900e04df)
(I'm guessing the two images have a different theme applied, causing a difference in color, but I'm only concerned about the thin outline.)

## `chadrc` file
```lua
---@type ChadrcConfig
local M = {}

-- Path to overriding theme and highlights files
local highlights = require "custom.highlights"

M.ui = {
  theme = "onedark",
  theme_toggle = { "onedark", "one_light" },

  hl_override = highlights.override,
  hl_add = highlights.add,

  telescope = { style = "bordered" }, -- this is the only change I made
}

M.plugins = "custom.plugins"

-- check core.mappings for table structure
M.mappings = require "custom.mappings"

return M
```
Running `:checkhealth telescope` returns the following:
![image](https://github.com/NvChad/NvChad/assets/27896672/19a0473c-d4b9-4249-af0e-fe9e2e5bfbad)

I am new to NeoVim and lua, so I am probably just missing something, but I can't seem to figure it out.

Thanks.

**Comment by Steven314** [comment, marked as answer]
Never mind. I closed my laptop and came back about an hour later to find it working properly. Not sure what went wrong.

**Reply by siduck** [reply]
@Steven314 you're supposed to save files with nvim only, cuz custom files* are reloaded & base46 cache is auto-compiled whenever you save a custom file within  nvim, using autocmds

**Reply by Steven314** [reply]
Ohhh. That makes sense. Yeah, I tested with my desktop (as opposed to my laptop), but I edited and saved the file in nvim, which worked. (I was previously using VSCode.)
Now seeing your reply clears things up. Thank you for the help.
